sqlmesh/core/engine_adapter/mssql.py

```diff
@@ -188,15 +191,15 @@ def _get_data_objects(
             FROM {catalog_name}.INFORMATION_SCHEMA.TABLES
             WHERE TABLE_SCHEMA LIKE '%{schema_name}%'
         """
-        dataframe: pd.DataFrame = self.fetchdf(query)
```

Comment:
why did you get rid of this? why don't you just fix fetchdf with a transaction wrap?